compliance-monitor/monitor.py

```diff
@@ -413,6 +417,11 @@ async def post_report(
     if content_type not in ('application/x-signed-yaml', 'application/x-signed-json'):
         # see https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/415
         raise HTTPException(status_code=415, detail="Unsupported Media Type")
+
+    with conn.cursor() as cur:
+        keys = db_get_keys(cur, account[0])
```

Comment:
`account[0]` is used in several places and seems to refer to account name/subject name (?), might be good for readers to have it saved in a dedicated variable `subject_name`.

The type in the function signature also doesn't give more information about the meaning, it just says `tuple[str,str]`

Reply:
9f10620785229d67c4f6cc39b303fd829445d4d3